Updating date and removing old IPs

vkonev opened the pull request:
We have decommissioned a lot of older agents. Removing old IPs from the list and updating date stamp

azurecla commented:
Hi __@leylakazemi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

mollybostic commented:
changed tag to indicate microsoft contributor

vkonev committed:
Updating date and removing old IPs

We have decommissioned a lot of older agents. Removing old IPs from the list and updating date stamp
